### PR TITLE
recover back AuthIdentityEvent

### PR DESCRIPTION
**Pull request by klizhentas**
Recover identity event used by Gravity

### CODE REVIEW

**Comment by kontsevoy** [comment]
retest this please

File: lib/service/service.go [hunk 1]
```diff
@@ -346,7 +348,25 @@ func (process *TeleportProcess) initAuthService(authority auth.Authority) error
 
 	// Heart beat auth server presence, this is not the best place for this
 	// logic, consolidate it into auth package later
-	var authClient *auth.TunClient
+	storage := utils.NewFileAddrStorage(
```
Comment (kontsevoy) [inline lib/service/service.go]: there's `TeleportProcess.connectToAuthServer()` which does exactly what this block of code does.

Comment (klizhentas) [inline lib/service/service.go]: right, but we don't call it for auth server anywhere.

Comment (kontsevoy) [inline lib/service/service.go]: it does what you need. no reason for code duplication. tun clients are messy and leaky, there's a function which creates one for you (and checks for connectivity to boot!)

Comment (kontsevoy) [inline lib/service/service.go]: maybe you're confusing it with "registerWithxxxx"?

**Pull request merged**